libs/contracts/README.md

```diff
@@ -1,3 +1,19 @@
-# FeatureBoard API Contracts
+# FeatureBoard Contracts
 
-Contains TypeScript type definitions for the WebSocket API message contracts of https://featureboard.app
+Contains TypeScript type definitions for FeatureBoard SDK
```

Comment:
Maybe a small example of the sort of types in here and maybe a `## Why` heading.

The why is ensuring that the client and server SDKs have common interfaces for universal JavaScript application frameworks like Remix and NextJS